x-pack/docs/en/security/backup-security-configuration.asciidoc

```diff
@@ -0,0 +1,240 @@
+
+* Snapshot the `.security` index in a dedicated repository, where read and write
+access is strictly restricted and audited.
+* If there are indications that the snapshot has been read, change the passwords
+of the users in the native realm and revoke API keys.
+* If there are indications that the snapshot has been tampered with, do not
+restore it. There is currently no option for the restore process to detect
+malicious tampering.
+
+The same recommendations apply to storing backups for configuration files.
+However, use special caution when you store the `elasticsearch.keystore` and
+private key file backups. Currently, the secure store is *not* password
+protected and any read access can compromise the passwords of external systems
+and private keys.
+
+[float]
+[[restore-security-configuration]]
+==== Restoring security configuration
+
+NOTE: You can restore a snapshot of the `.security` index only if it was created
+in the same major release version. The last minor version of every major release
+can convert and read both versions of the index.
+
+To restore your security configuration from a backup, log in to one of the node
+hosts, navigate to {es} installation directory, and follow these steps:
+
+. Make sure the repository holding the `.security` snapshot is installed.
++
+--
+For example:
+[source,shell]
+--------------------------------------------------
+GET /_snapshot/my_backup
+--------------------------------------------------
+--
+
+. Add a new user with the `superuser` built-in role to the file based realm.
++
+--
+For example, create a user named `jacknich`:
+[source,shell]
+--------------------------------------------------
+ bin/elasticsearch-users useradd jacknich -p password -r superuser
+--------------------------------------------------
+--
+
+. Using the previously created user, delete the existing `.security-6` or
+`.security-7` index.
++
+--
+[source,shell]
+--------------------------------------------------
+curl -u jacknich-X DELETE "localhost:9200/.security-7"
+--------------------------------------------------
+
+WARNING: After this step authentication will not work; all API calls will fail.
+
+--
+
+. Using the same user, restore the `.security` index from the snapshot.
++
+--
+[source,shell]
+--------------------------------------------------
+ curl -u jacknich -X POST "localhost:9200/_snapshot/my_backup/snapshot_1/_restore" -H 'Content-Type: application/json' -d'
+ {
+    "indices": ".security-7",
+    "include_global_state": true
+ }
+ '
+--------------------------------------------------
+
+NOTE: Restoring the global state is optional, but it will help make sure the
+
+--
+
+. Copy the backup configuration details by overwriting the contents of
+`$ES_PATH_CONF` and restart the node.
++
+--
+This step is common to every other backup restore.
+--
+
+. Cherry-pick and
+<<cluster-update-settings,apply the persistent secure settings>> from earlier.
```

Comment:
I think this step needs more information since it wasn't clear to me what "earlier" was referring to or what exact steps I needed to perform here.